### DISCUSSION TITLE
IEA 15MW wind turbine: Some vortices were found below the ground.

### DISCUSSION THREAD

**Discussion by ZL6462** [OpenFAST/openfast#1857] [General]
Dear all,

These days, I am configuring the IEA 15MW case with the OLAF method. The case I use come from "https://github.com/IEAWindTask37/IEA-15-240-RWT/tree/master/OpenFAST/IEA-15-240-RWT-OLAF", and I make the following modifications:
1) Considering no floating platform, namely the turbine I simulated is a fixed bottom wind turbine;
2) To solving the problem of a too high rotor power, the relavant parameters were adjust according "https://forums.nrel.gov/t/iea-15-mw-rwt-fixed-foundations-discrepancy-in-rtaeropwr-among-olaf-and-bemt/3411":

Using the case mentioned above, the program prompted that "some vortices were found below the ground". And through the visualization of the wake, some of the vortex lines do enter the ground， see the figure below:

![ac7bfe511c7812de452dc86b322a8ab](https://user-images.githubusercontent.com/98078424/187583048-dd899648-b0cb-40ad-9f52-d107337d356a.png)

How can I avoid such problem? Or is there any method the turn off the effect of the ground?
 
Thank you for your attention.

Best regards,

**Comment by 503283104** [comment]
Hello, 

I am new in the use of OLAF and I am facing the same issue. How did you managed to solve it? And how can I plot the wake?

Thanks in advance.

Best regards,

**Comment by fnaos** [comment]
Any updates on that? @ZL6462. I am also facing the same issue with the same model.

**Comment by ebranlard** [comment]
OLAF will provide this warning when vortices are present below z=0. OLAF will then bounce-back these vortices above the ground. Unfortunately, wake expansion and turbulence will often result in vortices entering the ground, unless a full account of the ground boundary is taken into account. We have no plan to account for that at the moment. 

You can use the "mirror" option of OLAF, which should introduce a non slip boundary condition at the ground, but at the expense of doubling the computational time. The method is also imprecise and might still lead to vortices bouncing in and out of the ground boundary. 

If you are running simulations wihtout shear, you can simply increase the hub height to avoid this issue. 

Otherwise, simply ignore these warnings, but check that the wake has a reasonable shape, for instance, use `WrVTK=2`, which  should have OLAF write the wake in the vtk_fvw folder at the end of the simulation, and then inspect the VTk file with Paraview. In the example figure provided above, the wake is altered and entering the ground about a diameter downstream of the turbine, which should likely not affect the loads at the turbine too much. 

The example provided above seem to use older guidelines and version of OpenFAST. Current guidelines do not recommend using a "far wake", but instead recommend using a near wake with a frozen region. This should also limit the number of vorticies entering the ground.
